src/main_scene.py

```diff
@@ -43,7 +43,7 @@ def show_objective(self):
 
     def show_total_saving(self):
```

Comment:
Since this function only run self.set_icon_size_pos you can just remove or replace this function and refactor dependencies on procedures which call this function.